src/meta/src/controller/catalog/drop_op.rs

```diff
@@ -319,6 +341,17 @@ impl CatalogController {
         inner
             .dropped_tables
             .extend(dropped_tables.map(|t| (TableId::try_from(t.id).unwrap(), t)));
+
```

Comment:
Do we need to notify the update if the table is also dropped together?

Reply:
Oh, never mind. Only table `!removed_streaming_job_ids.contains(&target_table_id)` will be added to `removed_sink_by_table`.